src/auth/ntlm/fake/ntlm_fake_auth.cc

```diff
@@ -50,6 +51,7 @@
 #if HAVE_GETOPT_H
 #include <getopt.h>
 #endif
+#include <thread>
```

Comment:
I am worried that including `<thread>` will require linking with the OS thread library on some platforms, even though we are not actually using any threads. I do not think that linking risk is worth the convenience of using sleep_for(). Using thread-named API will also confuse some readers into thinking that the helper is threaded.

I do not insist on refactoring this to avoid <thread>.

Reply:
Understood, though the OS we support with BuildFarm all show the helper building fine as-is.

Reply:
> OS we support with BuildFarm all show the helper building fine

That is good but probably irrelevant because the errors, if any, manifest at _runtime_, and because GCC/clang builds on popular platforms are unlikely to cause these issues. See https://stackoverflow.com/questions/8649828/what-are-the-correct-link-options-to-use-stdthread-in-gcc-under-linux for some of the similar examples (but not caused by sleep_for()).